examples/echo/tcp-echo/client/app/client.go

```diff
@@ -0,0 +1,164 @@
+/******************************************************
+# DESC    : echo client
+# AUTHOR  : Alex Stocks
+# LICENCE : Apache License 2.0
+# EMAIL   : alexstocks@foxmail.com
+# MOD     : 2016-09-06 17:24
+# FILE    : client.go
+******************************************************/
+
+package main
+
+import (
+	"math/rand"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+import (
+	"github.com/AlexStocks/getty"
```

Comment:
change to use dubbogo/getty

Reply:
Hey, guy, this is AlexStocks/getty. If this feature is merged into AlexStocks/getty, I will issue a new pr to dubbogo/getty.

Reply:
awkward